fixtures/src/components/uncontrolled/date-range.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import DateRange from 'carbon-react/lib/__experimental__/components/date-range';
+import { LogConsumer } from '../log';
+
+const UncontrolledDateRange = () => {
+  return (
+    <LogConsumer>
+      {(log) => {
+        const onChange = e => log(e, { method: 'onChange' });
+
+        const onBlur = e => log(e, { method: 'onBlur' });
+
+        return (
+          <React.Fragment>
+            <div id='uncontrolled_date_range'>
+              <h1>Uncontrolled Date Range</h1>
+              <ul>
+                <li>onChange handler should update the log when the value is changed, e.target.value should be an array
+                  that contains the &#123; formattedValue, rawValue &#125; the Date input&apos;s
+                </li>
+                <li>onBlur handler should update the log when the date is blurred
+                </li>
+                <li>date range has props value, name, id which should be reflected in both events</li>
```

Comment:
There's no `value` prop being passed to `<DateRange>` here.
```suggestion
                <li>date range has props name, id which should be reflected in both events</li>
```

Reply:
Thanks for fixing this in 68630c9 :white_check_mark: